tests/integration/conftest.py

```diff
@@ -31,7 +31,7 @@ def pytest_addoption(parser):
         '--work-dir', default='tests/tmp',
         help='Working directory for tests temporary files.')
     parser.addoption(
-        '--redis-up-timeout', default=120,
+        '--redis-up-timeout', default=3,
```

Comment:
Revert the change we made in https://github.com/RedisLabs/redisraft/pull/599

Some tests expect a node "down" after start-up. So, it is better to keep this value small, otherwise those tests take a long time.